src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -104,10 +105,7 @@ class AppDelegate: FlutterAppDelegate { // More on the FlutterAppDelegate.
 {% endtab %}
 {% tab "UIKit-ObjC" %}
 
-In this example, we create a `FlutterEngine` 
-object inside a SwiftUI `ObservableObject`. 
-We then pass this `FlutterEngine` into a 
-`ContentView` using the `environmentObject()` property.
+As an example, we demonstrate creating a FlutterEngine, exposed as a property, on app startup in the app delegate.
```

Comment:
This section is for UIKit-Swift, but we accidentally put SwiftUI content here. So I simply copied content from UIKit-ObjC.